bridges/pangolin-ropsten/bridge/src/service/redeem/mod.rs

```diff
@@ -36,86 +35,33 @@ impl Service for RedeemService {
         let tracker = Tracker::new(microkv, "scan.ropsten.redeem");
 
         // Receiver & Sender
-        let mut rx = bus.rx::<ToRedeemMessage>()?;
-        let sender_to_redeem_scan = bus.tx::<ToRedeemMessage>()?;
-        let mut sender_to_redeem_command = bus.tx::<ToRedeemMessage>()?;
-        let sender_to_extrinsics_command = bus.tx::<ToExtrinsicsMessage>()?;
         let sender_to_extrinsics_scan = bus.tx::<ToExtrinsicsMessage>()?;
 
         let _greet_scan = Self::try_task(
             &format!("{}-service-redeem-scan", PangolinRopstenTask::name()),
             async move {
                 start_scan(
                     tracker.clone(),
-                    sender_to_extrinsics_scan.clone(),
-                    sender_to_redeem_scan.clone(),
+                    sender_to_extrinsics_scan.clone()
                 )
                 .await;
                 Ok(())
             },
         );
 
-        let _greet_command = Self::try_task(
-            &format!("{}-service-redeem-command", PangolinRopstenTask::name()),
```

Comment:
Yes, we can remove this service. but we need to add custom command to support redeem. 
like this https://github.com/darwinia-network/bridger/blob/v0.4.13/task/task-pangolin-ropsten/src/route/redeem.rs
and the new command send extrinsic directly. will be create a new issue to track this.